### PR TITLE
Adding a processing list to not have two threads process the same work item.

### PR DESCRIPTION
**Pull request by irieksts**

### CODE REVIEW

File: src/main/java/com/hmsonline/cassandra/triggers/LogEntry.java [hunk 1]
```diff
@@ -132,4 +132,11 @@ public boolean hasErrors(){
         return (this.getErrors() != null && this.getErrors().size() > 0);
     }
```
Comment (boneill42) [inline src/main/java/com/hmsonline/cassandra/triggers/LogEntry.java]: Delete  this if it is not used.

File: src/main/java/com/hmsonline/cassandra/triggers/ProcessingManager.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+/**
+ * @author <a href=irieksts@healthmarketscience.com>Isaac Rieksts</a>
+ *
+ */
+public class ProcessingManager {
+  private static long TIME_TO_LIVE = 5000;
+  private ConcurrentHashMap<String, Long> processing;
+
+  public boolean check(String key) {
```
Comment (boneill42) [inline src/main/java/com/hmsonline/cassandra/triggers/ProcessingManager.java]: change the name of this method to "isAlreadyBeingProcessed" or something a bit more informative.
(it's unclear what it is doing down when it is used)

File: src/main/java/com/hmsonline/cassandra/triggers/ProcessingManager.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
+package com.hmsonline.cassandra.triggers;
+
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * @author <a href=irieksts@healthmarketscience.com>Isaac Rieksts</a>
+ *
+ */
```
Comment (boneill42) [inline src/main/java/com/hmsonline/cassandra/triggers/ProcessingManager.java:8]: You might also consider putting the blocking queue into this class as well.
(not sure on this one, but if you put both in here you would encapsulate everything)

Comment (irieksts) [inline src/main/java/com/hmsonline/cassandra/triggers/ProcessingManager.java:8]: Yes, lets come back and refactor.

**Pull request merged**